### ISSUE TITLE
Screen reader is not announcing Name, Role and value for controls present in 'Example tab' using tab key keys in Windows.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#2105]
**Describe the bug**
Screen reader is not announcing Name, Role and value for controls present in 'Example tab' using tab key keys in Windows. When the screen reader does not announce the Name, Role and State of controls, it can lead to confusion and difficulty in navigating the application. Users may not be able to understand the purpose of certain controls or how to interact with them effectively, which can hinder their ability to use the app independently.

**To Reproduce**

1. Open '[Blazor CheckBoxList Component | Free UI Components by Radzen](https://blazor.radzen.com/checkboxlist?theme=material) in latest Edge browser.
2. Checkbox list page will appear.
3. Press tab key to reach 'order, Employment' Checkboxes present in Example Tab.
4. Verify using tab key whether screen reader is announcing Name, Role and State for controls present in 'Example tab' or not

Alternatively link your repo with a sample project that can be run.

**Expected behavior**
Screen reader should announce Name, Role and value for controls present in 'Example tab' using tab key keys in Windows.
It should announce as 'Example grouping list with 3 items orders check box not checked.'

**Screenshots**

https://github.com/user-attachments/assets/9a012a42-bb05-4d8c-b745-2e5a427d49f8

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/42b47208-2d9e-4396-b3a4-b9ce6e68ba1c" />

**Desktop (please complete the following information):**
OS: Windows 11 Version 24H2(OS Build 26100.3194)
Browser: Edge Version 133.0.3.65.82
URL: [Blazor CheckBoxList Component | Free UI Components by Radzen](https://blazor.radzen.com/checkboxlist?theme=material)
Screen reader: NVDA (2024.4.1.34550), JAWS and Narrator

**Additional context**

**Comment by enchev** [comment]
Improved in our 7.0 branch - will be released later this week or early next week.

**Comment by msftedad** [comment]
@enchev , Thank you for the update. Will check it in 7.0 branch once it is released.

**Comment by msftedad** [comment]
@enchev , Issue is still repro'ing at our end. Can i know if the version 7.0 branch is released and if not can you please let us know when will it releases?
